src/kbmod/reprojection.py

```diff
@@ -624,6 +790,56 @@ def _reproject_images(science_images, variance_images, mask_images, obstime, com
     return science_add, variance_add, mask_add, obstime
 
 
+def _write_images_to_shard(
+    science_add, variance_add, mask_add, psf, wcs, obstime, obstime_index, indices, directory, filename
+):
+    """
```

Comment:
Can you add a one sentence summary of the function? In this case I'm guessing is something along the lines of "Write the three image layers to a workunit shard"